homeassistant/components/ibeacon/device_tracker.py

```diff
@@ -3,8 +3,7 @@
 
 from ibeacon_ble import iBeaconAdvertisement
 
-from homeassistant.components.device_tracker import SourceType
-from homeassistant.components.device_tracker.config_entry import BaseTrackerEntity
+from homeassistant.components.device_tracker import BaseTrackerEntity, SourceType
```

Comment:
This integration shouldn't use the base class directly. We should add a new tracker class for BLE I think.

Reply:
Indeed, only ibeacon uses BaseTrackerEntity
I have reverted that change, and ibeacon can be adressed in a separate PR.